examples/demo-rollup/rollup_config.toml

```diff
@@ -4,7 +4,7 @@ start_height = 1
 
 [da]
 # The JWT used to authenticate with the celestia light client. Instructions for generating this token can be found in the README
-celestia_rpc_auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.jp_hghhRkcur3g0o-QhQnkLt89YPu4oykLtZW52_ZA0"
+celestia_rpc_auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.i6Cx-WziFU3Jcz2INKii2uMCareIZHDDMR0v8zW5zZc"
```

Comment:
I think this should be set to `"MY_SECRET_TOKEN"`, not an actual token, right?

Reply:
We have an issue for this, so I will let @dubbelosix handle it: https://github.com/Sovereign-Labs/sovereign-sdk/issues/481